### ISSUE TITLE
Opening github from the toolbar doesn't open after closing pane

### ISSUE THREAD

**Issue by ungb** [atom/github#891]
### Prerequisites

* [x] Have you tried to reproduce the problem in [Safe Mode][safe-mode]?
* [x] Have you followed all applicable steps in the [debugging guide][debugging-guide]?
* [x] Have you checked the FAQs on the [message board][message-board] for common solutions?
* [x] Have you checked that your issue [isn't already filed][already-filed]?

[safe-mode]: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
[debugging-guide]: http://flight-manual.atom.io/hacking-atom/sections/debugging/
[message-board]: https://discuss.atom.io/c/faq
[already-filed]: https://github.com/atom/github/issues

### Description

Opening Github or Git Pane after closing the pane adds the pane back to the dock, but doesn't open the dock until you click again.

### Steps to Reproduce

1.  Open Atom.
2. If github package is open, close it by right clicking and clicking `close pane`
3. On the bottom right click on the `[0-9]+ files`

**Expected behavior:**
[What did you expect to happen?]
Opens the git panel

**Actual behavior:**
[What actually happened instead?]
Nothing happens or it seems that way, but actually the git panel gets added to the dock on the right but it's closed.

 
**Reproduces how often:**
[What percentage of the time does this happen?]
100%

### Versions

You can get this information from copy and pasting the output of `atom --version` and `apm --version` from the command line. Also, please include the OS and what version of the OS you're running.
```
Atom    : 1.19.0-dev-58313f6ab
Electron: 1.6.9
Chrome  : 56.0.2924.87
Node    : 7.4.0
```
### Additional Information
gif:

![pane](https://cloud.githubusercontent.com/assets/5624870/26608233/5c973f5e-454f-11e7-87c2-1fe173538433.gif)

Please include any additional information, configuration or data that might be necessary to reproduce the issue. Screenshots, if appropriate, are helpful as well.

If your issue involves a repository, and that repository is public, please include a link to it.

I noticed this while testing fix for https://github.com/atom/github/issues/872
/cc @smashwilson

**Comment by smashwilson** [comment]
This is weird: I can reproduce, but it shows fine on the _second_ click?

**Comment by smashwilson** [comment]
Okay, I think I got it :smile: It's published in 0.3.1 and included in Atom master and beta. Care to give it another spin?

**Comment by ungb** [comment]
LGTM! Awesome job on this @smashwilson a lot smoother experience now that it only takes 1 click instead of two to open the git/github panel :D
